testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -659,7 +659,7 @@
                               ~ Used for Wildfly Elytron 1.13.0.CR3+ RESTEasy client SSL truststore configuration.
                               ~ See KEYCLOAK-15692, ELY-1891 issues & PRs of EAP7-1219 issue for details.
                               -->
-                            <wildfly.config.url>${project.build.directory}/dependency/wildfly-config.xml</wildfly.config.url>
+                            <path.to.wildfly.config>${project.build.directory}${file.separator}dependency${file.separator}wildfly-config.xml</path.to.wildfly.config>
```

Comment:
Why was this property renamed? Can the name remain as was? `wildfly` is the most determining part of the property and should remain first.

Reply:
Hi @hmlnarik, Jan explained it in the [comment](https://github.com/keycloak/keycloak/pull/7852#issuecomment-798887126) above. The reason was to avoid a naming collision with wildfly-config-client, which uses a property with the same name but it parses it differently.

I guess we could use something like `<wildfly.config.path>` instead.

Reply:
Thanks @Pepo48. Would `wildfly-client.config.path` work any better?

Reply:
Thanks @hmlnarik `wildfly-client.config.path` LGTM

(previously, based on the current content of that `wildfly-config.xml` file wanted to suggest something like `wildfly.elytron.client.config.path`, but given the content of that file might be enhanced in the future yet possibly with other options from [the allowed syntax](https://docs.wildfly.org/23/Client_Guide.html), it's better to use more general purpose name)

For your previous questions:
- We can't use `wildfly.config.url` directly due the reasons in comment \#0 (description) of this PR,
- And `path.to.wildfly.config` was chosen just to avoid a name, analogous to `wildfly.config.url`, which potentially could lead to re-introduction of the same issue (attempt to re-use `wildfly.config.xml`) again.

But `wildfly-client.config.path` LGTM. @Pepo48 What's our take on this?

Reply:
Used `wildfly-client.config.path` with the latest state / change.

Reply:
@iankko I'm also fine with proposed. Thanks!